### PR TITLE
Fix kurbo::BezPath d attribute for xilem_web::svg::path element

### PR DESCRIPTION
**Pull request by RagibHasin**
Regressed in #699

cc @Philipp-M 🦆

### CODE REVIEW

**Review by xorgy — APPROVED** [review]

File: xilem_web/src/svg/kurbo_shape.rs [hunk 1]
```diff
@@ -213,7 +213,7 @@ impl<State: 'static, Action: 'static> OrphanView<BezPath, State, Action, DynMess
             if attrs.was_created() {
                 attrs.push(("path", new.to_svg()));
```
Comment (xorgy) [inline xilem_web/src/svg/kurbo_shape.rs:214]: Doesn't this also need changed?

**Review by xorgy — CHANGES_REQUESTED** [review]

**Comment by Philipp-M** [comment]
Whoops, not sure how that did happen...

Nice catch, I thought I tested this, but this is the only kurbo view that's not in the svgtoy example... Bad luck...

File: xilem_web/src/svg/kurbo_shape.rs (hunk 1 above)
Comment (RagibHasin) [reply xilem_web/src/svg/kurbo_shape.rs]: Yeah, I think so. BTW, it was this way before #699 too.

**Review by Philipp-M — APPROVED** [review]
Thanks!

File: xilem_web/src/svg/kurbo_shape.rs (hunk 1 above)
Comment (RagibHasin) [reply xilem_web/src/svg/kurbo_shape.rs]: Or maybe it hasn't been. I am too sleepy to think coherently.

**Comment by DJMcNab** [comment]
Should we have an example which uses this?

**Pull request merged**